### ISSUE TITLE
[BUG] - pas de données de la veille

### ISSUE THREAD

**Issue by floow77** [m4dm4rtig4n/myelectricaldata#173]
**Description du bug**
Depuis l'installation du module, je n'ai jamais réussi à récupérer les données de la veille.
Aucune erreur dans les mots

![image](https://user-images.githubusercontent.com/72973078/205558439-4633ce26-6a63-462e-95c9-67fa45afe337.png)

**Configuration**
Hassio
Myelectricaldata v0-8-8-dev

**Comment by m4dm4rtig4n** [comment]
J'ai également constaté le soucis. Je vais matter ca prochainement.

**Comment by vingerha** [comment]
J'ai testé 085-dev et ça marche .. mais en 0.8.6-dev non ... ni 087

**Comment by pbranly** [comment]
Tu n'es pas en dépassement de quota ?
J'ai le même problème mais je suis en super depassement !

**Comment by vingerha** [comment]
Non.... recups continue mais pas pour hier...085 passe le recup sans problème

**Comment by m4dm4rtig4n** [comment]
Alors pour le coup j'ai volontairement mit un delta d'1j sur la récup suite à un soucis, mais je ne sais plus lequel...

En gros si je demande de la donnée à Enedis entre le 2022-10-29 => 2022-12-04 (comme actuellement) Enedis ne m'envoie pas la consommation du 04.
Je viens de retirer le delta d'1j sur la 0.8.8 qui va pop prochainement en dev.

**Comment by vingerha** [comment]
Confirmé que ça marche avec 088-dev

**Comment by TeddyLafrite** [comment]
Bonsoir

Toujours le même problème pour moi avec la stable 0.8.8 après MAJ ce matin

**Comment by vingerha** [comment]
Bjr, ce matin les données ont arrivés tard...t'as encore des soucis? Moi j'avais 088-dev et maintenant 088 ...les deux marchent bien

**Comment by TeddyLafrite** [comment]
Je viens de relancer l'appIdem et ça fait plusieurs jours que c'est comme ça, depuis le passage à myelectricaldataJ'ai quelques erreurs dans les logs je pense à des valeurs à 0 sur les tarifs HP HC (suis en base)Je vais regarder plus en détail si ça vient de là Le 10/12/2022, 13:39 Arjan ***@***.***> a écrit:

   
  Bjr, ce matin les données ont arrivés tard...t'as encore des soucis? Moi j'avais 088-dev et maintenant 088 ...les deux marchent bien
